tests/custom_id.test.ts

```diff
@@ -0,0 +1,90 @@
+import { objectType } from 'nexus'
+import { createRuntimeTestContext } from './__client-test-context'
+
+let ctx = createRuntimeTestContext()
+
+it('supports @id fields with custom name', async () => {
+  const datamodel = `
+    model User {
+      custom_id  Int @id @default(autoincrement())
+      posts Post[]
+    }
+
+    model Post {
+      custom_id  Int @id @default(autoincrement())
+      title String
+    }
+  `
+  const User = objectType({
+    name: 'User',
+    definition(t: any) {
+      t.model.custom_id()
+      t.model.posts()
+    },
+  })
+
+  const Post = objectType({
+    name: 'Post',
+    definition(t: any) {
+      t.model.custom_id()
+      t.model.title()
+    },
+  })
+
+  const Query = objectType({
+    name: 'Query',
+    definition(t: any) {
+      t.crud.users()
+    },
+  })
+
+  const { graphqlClient, dbClient } = await ctx.getContext({
+    datamodel,
+    types: [Query, User, Post],
+  })
```

Comment:
Not super happy with the `getContext` naming here but couldn't find anything better